galaxy/Dockerfile

```diff
@@ -19,7 +19,7 @@ GALAXY_CONFIG_DIR=/etc/galaxy
 ENV GALAXY_CONFIG_FILE=$GALAXY_CONFIG_DIR/galaxy.ini \
 GALAXY_CONFIG_JOB_CONFIG_FILE=$GALAXY_CONFIG_DIR/job_conf.xml \
 GALAXY_CONFIG_JOB_METRICS_CONFIG_FILE=$GALAXY_CONFIG_DIR/job_metrics_conf.xml \
-GALAXY_VIRTUALENV=$GALAXY_ROOT/.venv/ \
+GALAXY_VIRTUAL_ENV=/galaxy_venv \
```

Comment:
We actually have a few per-OS wheels (like Ubuntu is different CentOS) - it wouldn't matter for anything running on the cluster though, but I have had problems with relocating virtualenvs before. I'll update this PR with a link to the Galaxy docs and a short discussion (though it will by hypothetical because I don't have a cluster).